Update PipelineRerunNIRCAM-LONG.py

SpacialTree opened the pull request:
Adding post-pipeline offset alignment to better align to catalog/refcat

SpacialTree committed:
Update PipelineRerunNIRCAM-LONG.py

Adding post-pipeline offset alignment to better align to catalog/refcat

SpacialTree commented:
Dont merge this yet

SpacialTree committed:
Update PipelineRerunNIRCAM-LONG.py

adding in star_coord, pix_coord dictionary, and some imports

SpacialTree commented:
Okay, now it should be fine to merge

keflavich commented on reduction/PipelineRerunNIRCAM-LONG.py:
```diff
@@ -61,6 +64,11 @@ def print(*args, **kwargs):
 # see 'destreak410.ipynb' for tests of this
 medfilt_size = {'F410M': 15, 'F405N': 256, 'F466N': 55}
 
+# Pixel coordinates of star
+pix_coords = {'nrca': (3904, 869), 'nrcb': (1119, 832), 'merged': (3903, 868)}
+#Sky coordinate of star
+star_coord = SkyCoord(266.594893*u.deg, -28.587417*u.deg)
+
```
since this stuff is only for cloud c, can you wrap it up so the code recognizes that it's only for cloud c / 2221?

https://github.com/keflavich/brick-jwst-2221/blob/main/reduction/destreak.py#L10-L21 gives an example of what that might look like

SpacialTree replied:
Done

SpacialTree committed:
Update PipelineRerunNIRCAM-LONG.py

Wrapping bulk offset code so that it only applies to the proposal / field applicable.
